index.bs

```diff
@@ -281,6 +281,12 @@ The <dfn method for=IdleDetector>requestPermission()</dfn> method steps are:
 
 The <dfn method for=IdleDetector>start(|options|)</dfn> method steps are:
 
+1.  If the [=relevant global object=] of [=this=] is a {{Window}} object whose
+    [=associated Document=] is not [=fully active=], return [=a promise rejected with=]
```

Comment:
The equivalent worker concept is "[active needed worker](https://html.spec.whatwg.org/multipage/workers.html#active-needed-worker)". You're right that the code below assumes a `Document` because the Permissions Policy spec does not currently define how it works in workers. In Chromium, since this API is only available in dedicated workers, the permissions policy is simply inherited from the owning document.